gramps/gui/selectors/baseselector.py

```diff
@@ -315,8 +315,14 @@ def build_tree(self):
         """
         Builds the selection people see in the Selector
         """
-        if not self.filter[1]:
-            filter_info = (False, self.search_bar.get_value(), False)
+        # any search_bar filter takes precedence over self.filter
```

Comment:
I'm not sure if we want this.  Why should a search take precedence over a filter?

Reply:
I agree. PR #1865 does just that.
I'm tempted to cherry-pick the first commit (which is a bug fix) into PR #1865 and abandon this one.
Can you let me know if you agree?

Reply:
Yes.  We can still keep it as a separate commit.